src/PBot/TaskForces/InvolvedIssueQuery.cs

```diff
@@ -37,13 +37,22 @@ public InvolvedIssueQuery(GitHubClient client)
         {
             var results = new List<InvolvedIssue>();
 
-            var filter = new RepositoryIssueRequest
+            var mentionedFilter = new RepositoryIssueRequest
             {
                 State = ItemState.Open,
                 Mentioned = username
             };
 
-            var issues = await client.Issue.GetForRepository("Particular", repo.Name, filter);
+            var assigneeFilter = new RepositoryIssueRequest
+            {
+                State = ItemState.Open,
+                Assignee = username
+            };
+
+            var issues = (await client.Issue.GetForRepository("Particular", repo.Name, mentionedFilter))
```

Comment:
This will run the queries in sequence, slowing down the call. Is it worth trying to run these in parallel?